src/server.rs

```diff
@@ -433,6 +413,33 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn get_child_version_not_found_not_latest() -> anyhow::Result<()> {
+        init_logging();
+
+        let storage = InMemoryStorage::new();
+        let mut txn = storage.txn()?;
+        let client_id = Uuid::new_v4();
+
+        // add a parent version, but not the requested child version
+        let parent_version_id = Uuid::new_v4();
+        txn.new_client(client_id, parent_version_id)?;
+        txn.add_version(client_id, parent_version_id, NIL_VERSION_ID, vec![])?;
```

Comment:
This line appears to be the only functional difference between this test and `get_child_version_gone`. it isn't clear to me what difference we'd expect between them.

Reply:
I think you're right!